About/whitepaper.md

```diff
@@ -22,23 +22,23 @@ sub: '
   </a>
 </div>
 
-seL4 is a high-assurance, high-performance operating system microkernel. It is
-unique because of its comprehensive formal [verification](../Verification/),
-without compromising [performance](Performance/). It is meant to be used as a
+seL4 is a high-assurance, high-performance operating system microkernel. Its
+uniqueness lies in the comprehensive formal [verification](../Verification/),
+while maintaining high [performance](Performance/). seL4 was designed as a
```

Comment:
Happy with the change for the second sentence, but the first sentence should stay as is. `It is unique` is the main message of the first part of that sentence and `its uniqueness lies in` makes the uniqueness a side issue. `compromising performance` is what (some) people are afraid of. `Maintaining high performance` assumes that something already has high performance, but for microkernels that is not a given.

Reply:
I also prefer not changing that 1st sentence for the reasons Gerwin mentions. In the second sentence, I would have present tense "seL4 is designed as.." (else it could be interpreted as it's not true anymore...)

Reply:
Fair enough. I found the crux of first sentence "It is unique without compromising performance" a little unclear. When I read it, I questioned why an unique technology would compromise performance. (Nit-picking, I know.  :-))

Reply:
> I also prefer not changing that 1st sentence for the reasons Gerwin mentions. In the second sentence, I would have present tense "seL4 is designed as.." (else it could be interpreted as it's not true anymore...)

"seL4 is designed as.." (else it could be interpreted as it's not true anymore...)
I have the usual issue with past/present tense here. I.e. I don't necessarily agree that "seL4 is designed as" equates to "seL4 has already been designed, and continues to be designed"
(I'm not suggesting writing the latter one, but I think it's what you're saying.)
What about saying "seL4 is a trustworthy foundation for building safety- and security-critical systems."?
It's factual and avoids the tense issue.

Reply:
> seL4 is a trustworthy foundation for building safety- and security-critical systems. 

👍 Let's use that one.

I need to re-read the paragraph in context to make sure it still flows, but it fixed the issue that was there before, and we can still tweak more later if it turns out it is needed.

Reply:
Turns out that did not work well in the flow, because we have too many sentences with "seL4 is " already and the way we use them it would at least need a new paragraph. I think "is designed" is not great, but acceptable. It's passive, so it doesn't say when it was designed, it just describes the current state. And the tense does fit the rest.